src/docdb/tree/DocDBStoredProcedureTreeItem.ts

```diff
@@ -36,4 +39,19 @@ export class DocDBStoredProcedureTreeItem implements IAzureTreeItem {
             dark: path.join(__filename, '..', '..', '..', '..', '..', 'resources', 'icons', 'dark', 'Process_16x.svg')
         };
     }
+
+    public async deleteTreeItem(_node: IAzureNode): Promise<void> {
+        const message: string = `Are you sure you want to delete collection '${this.label}' and its contents?`;
+        const result = await vscode.window.showWarningMessage(message, DialogBoxResponses.Yes, DialogBoxResponses.Cancel);
+        if (result === DialogBoxResponses.Yes) {
+            const client = getDocumentClient(this._documentEndpoint, this._masterKey, this._isEmulator);
+            await new Promise((resolve, reject) => {
+                client.deleteStoredProcedure(this.link, function (err) {
+                    err ? reject(err) : resolve();
```

Comment:
For a destructive operation like this, would it make sense to write the result of this operation to the output,?

Reply:
Don't think we're doing it for anything else like that.  Will check.

Reply:
@fiveisprime Do you want us to print to the output window for every action?  We're currently doing it for deleting accounts but nothing else I think.